lib/riscv64gc-unknown-none-elf.json

```diff
@@ -0,0 +1,24 @@
+{
+	"arch": "riscv64",
+	"code-model": "medium",
+	"cpu": "generic-rv64",
+	"data-layout": "e-m:e-p:64:64-i64:64-i128:128-n64-S128",
+	"eh-frame-header": false,
+	"emit-debug-gdb-scripts": false,
+	"features": "+m,+a,+f,+d,+c",
+	"is-builtin": false,
+	"linker": "rust-lld",
+	"linker-flavor": "ld.lld",
+	"llvm-abiname": "lp64d",
+	"llvm-target": "riscv64",
+	"max-atomic-width": 64,
+	"panic-strategy": "abort",
+	"relocation-model": "static",
+	"target-pointer-width": "64",
+	"pre-link-args": {
+		"ld.lld": [
+			"-Triscv64/src/kernel.ld",
+			"-nostdlib"
+		]
+	}
+}
```

Comment:
It looks like newlines are missing at the end of this (and several other?) files.